src/bio/bio_device.c

```diff
@@ -373,6 +366,16 @@ replace_dev(struct bio_xs_context *xs_ctxt, struct smd_dev_info *old_info,
 	D_ASSERT(owner_thread(bbs) != NULL);
 	spdk_thread_send_msg(owner_thread(bbs), setup_bio_bdev, old_dev);
 
+	/* Set the LED of the VMD device to OFF state */
+	rc = bio_led_manage(xs_ctxt, NULL, old_dev->bb_uuid, (unsigned int)CTL__LED_ACTION__SET,
+			    &led_state, 0);
+	if (rc != 0) {
+		if (rc != -DER_NOSYS)
```

Comment:
Might be worth a comment on why this code is acceptable but others aren't.

Reply:
done